### DISCUSSION TITLE
de-opkomst-van-ai

### DISCUSSION THREAD

**Discussion by fjkresin** [fdnd/docs.fdnd.nl#273] [Reflectie]
Hoeveel studenten heb je die desondanks 'doorschieten' in het gebruik van GenAI en hoe ga je daarmee om? De snelle opkomst van GenAI kon in 2019/2020 nog niet worden voorzien; wat heeft dit betekend voor het curriculum? Betekent het ook iets voor de eindtermen en de rubrics?

**Comment by ju5tu5** [comment]
Thanks! Ik heb de tekst aangescherpt en de impact op toetsing aangestipt (die is er niet want wij beoordelen begrip inzicht en geen producten)
